### DISCUSSION TITLE
Chrome 121

### DISCUSSION THREAD

**Discussion by krausest** [krausest/js-framework-benchmark#1587] [Announcements]
Chrome 121 results are published: https://krausest.github.io/js-framework-benchmark/2024/table_chrome_121.0.6167.85.html

This time (but it'll be only this time) I also ran the benchmark on my old windows machine:
https://krausest.github.io/js-framework-benchmark/2024/table_chrome_121.0.6167.86_win.html

Attached you find a binary build of all frameworks such that you can run the benchmark without installing and building all frameworks (see README for how to use the build.zip)

No big changes from chrome 120.

<hr /><em>This discussion was created from the release <a href='https://github.com/krausest/js-framework-benchmark/releases/tag/chrome121'>Chrome 121</a>.</em>

**Comment by twharmon** [comment]
I think your link is for Chrome 120.

**Comment by WebReflection** [comment]
> No big changes from chrome 120.

shaving my yard here ... but uhtml is a way better fit than last time I've filed an update PR ... now I am curious to know if I'm prematurely happy about it, or actually the link is indeed an older one I've never had a change to overview 🤔

**Reply by krausest** [reply]
I trust full runs more than the incremental updates. The incremental runs mix different minor versions and I guess that some security fixes have an impact on performance.

**Reply by krausest** [reply]
I just could upload [windows results ](https://krausest.github.io/js-framework-benchmark/2024/table_chrome_121.0.6167.86_win.html)for chrome 121. uhtml score pretty similarly there.

**Reply by WebReflection** [reply]
ha ... lol, it's apparently the fastest ever first paint in there, what an achievement 🥳

**Comment by krausest** [comment]
Thanks. I had noticed the wrong link and changed it in the release, but not in this discussion.
